Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2922,15 +2922,7 @@ extension BrowserViewController: PreferencesObserver {
              Preferences.Shields.fingerprintingProtection.key:
             tabManager.allTabs.forEach { $0.webView?.reload() }
```

Comment:
Shouldn't we stop reloading tabs for other global settings too?

Reply:
We should ideally, we could reuse the same reset() for all such cases. Should I do it in this PR itself?

Reply:
No, please add a ticket for it, or maybe there;s something added already, like https://github.com/brave/brave-ios/issues/825

Reply:
great